shallow_backup/utils.py

```diff
@@ -25,11 +25,23 @@ def run_cmd_write_stdout(command, filepath):
 	"""
 	Runs a command and then writes its stdout to a file
 	:param: command str representing command to run
+	:param: filepath str file to write command's stdout to
 	"""
 	process = run_cmd(command)
-	if process:
+	if process and process.returncode == 0:
 		with open(filepath, "w+") as f:
 			f.write(process.stdout.decode('utf-8'))
+	else:
+		print_path_red("An error occurred while running: $", command)  # skip package or say it's not installed?
+		return 1
+
+
+def new_dir_is_valid(abs_path):
+	if os.path.isfile(abs_path):
+		print_path_red('New path is an existing file:', abs_path)
```

Comment:
```suggestion
		print_path_red('New path is a file:', abs_path)
```

Maybe this makes it more clear that the error is due to the path being a file, and not that it already exists? I could see some confusion stemming from that.

Reply:
6c7f9fadd91c616a78171d43564e4234f1cb4f1c